src/pages/add-lock-exception/AddLockExceptionForm/OrganisationUnitSelection/Controls.js

```diff
@@ -0,0 +1,68 @@
+import i18n from '@dhis2/d2-i18n'
+import {
+    CircularLoader,
+    SingleSelectField,
+    SingleSelectOption,
+    ButtonStrip,
+    Button,
+} from '@dhis2/ui'
+import PropTypes from 'prop-types'
+import React from 'react'
+import styles from './Controls.module.css'
+
+const Controls = ({
```

Comment:
The `Controls` component shared `SelectByLevel` and `SelectByGroup` is based on the dropdown/buttons shared by the [prior `d2-ui` iteration of these components](https://github.com/dhis2/d2-ui/blob/master/packages/org-unit-select/src/common.js#L43-L80).